tests/integration/test_outcome.py

```diff
@@ -115,6 +115,48 @@ def test_outcomes_custom_topic(
     assert start <= event_emission <= end
 
 
+def test_outcomes_two_configs(
```

Comment:
I just run this test on master and passes. Can we check the test again and verify the changes the PR introduces are correct?

Reply:
Test will pass on master, since we have only 1 Kafka cluster in integration tests, which is basically same `default`  config.
The test makes sure that we actually can use another configuration, and still can send the events , using that new config for that topic